examples/buttons/modules/buttons.js

```diff
@@ -0,0 +1,23 @@
+var buttonHandler = {};
+
+window.onPress = function onPress(type, callback) {
+  if (['back', 'up', 'select', 'down'].indexOf(type) >= 0) {
```

Comment:
@cat-haines asked me to do a simple implementation just with short press.

If you'd like me to do long press as well I can investigate how to do this but it wasn't part of the JIRA's initial scope I don't think.